routes/middleware/post_place.js

```diff
@@ -24,8 +25,11 @@ module.exports = (req, res, next) => {
   };
 
   request(options, (error, result, body) => {
+    /* istanbul ignore next */
     if (error) {
-      res.send(error);
+      res.render('home', {
+        title: 'Not found'
```

Comment:
You should probably add a log to all these errors so you can at least tell how things are going wrong on the backend.

Reply:
so i guess only in development im allowed to have console logs in the backend right ?

Reply:
but im testing for errors in my tests so my terminal would just get filled up with errors, is it still a good idea?

Reply:
I was actually thinking more along the lines of it being _good_ to have `console.log`s in production so there's a record of any errors.

Reply:
Not a blocker or particularly relevant to this PR...